rest/api.go

```diff
@@ -332,13 +334,16 @@ func (h *handler) handleHeapProfiling() error {
 		return err
 	}
 	err = pprof.WriteHeapProfile(f)
+
+	fileCloseError := f.Close()
```

Comment:
Should be checking fileCloseError below.

Reply:
Changes are committed to the latest revision.